test/unit/lib/api-client.js

```diff
@@ -244,6 +245,32 @@ describe('API Client', () => {
         });
     });
 
+    it('getModelAndItems', () => {
+        let response = {
+            status: 'success',
+            data: {
+                model: 'm8y7nEtAe',
```

Comment:
NACK: This doesn't match what the server responds with. https://github.com/TheFourFifths/consus/blob/25fcae2936a6795228acb40312ef1cad1f2d3966/docs/REST.md#get-apimodelchildren